galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager.ivt/src/main/java/dev/galasa/zos3270/manager/ivt/Zos3270IVT.java

```diff
@@ -46,8 +48,12 @@ public class Zos3270IVT {
     public String providedRunName;
     private String runName  = new String();
     
+	private String credentialsId = "PRIMARY";
     private String applid = "IYK2ZNB5";
 
+	@TestProperty(prefix = "IVT.REGION", suffix = "APPLID")
+	public String cbsaApplid;
+
     @Test
     public void checkInjection() {
         assertThat(logger).as("Logger Field").isNotNull();
```

Comment:
can we remove the isNotNull assertions

Reply:
Resolved in https://github.com/galasa-dev/managers/pull/818/commits/31ddf99e10cc7d6e738c9d00daccdf572ad412c0